src/image/manifest.rs

```diff
@@ -164,6 +164,11 @@ impl ImageManifest {
     pub fn to_writer_pretty<W: Write>(&self, writer: &mut W) -> Result<()> {
         to_writer(&self, writer, true)
     }
+
+    /// Access the image layers mutably.
+    pub fn get_layers_mut(&mut self) -> &mut Vec<Descriptor> {
+        &mut self.layers
+    }
```

Comment:
We can derive those by using [`MutGetters`](https://docs.rs/getset/latest/getset/derive.MutGetters.html): 

```patch
diff --git a/src/image/config.rs b/src/image/config.rs
index 6d6ec6c46..79aae8cce 100644
--- a/src/image/config.rs
+++ b/src/image/config.rs
@@ -175,11 +175,6 @@ impl ImageConfiguration {
     pub fn to_writer_pretty<W: Write>(&self, writer: &mut W) -> Result<()> {
         to_writer(&self, writer, true)
     }
-
-    /// Access the history mutably.
-    pub fn get_history_mut(&mut self) -> &mut Vec<History> {
-        &mut self.history
-    }
 }
 
 impl Default for ImageConfiguration {
diff --git a/src/image/manifest.rs b/src/image/manifest.rs
index d5b4f291f..71496117b 100644
--- a/src/image/manifest.rs
+++ b/src/image/manifest.rs
@@ -4,7 +4,7 @@ use crate::{
     from_file, from_reader, to_file, to_writer,
 };
 use derive_builder::Builder;
-use getset::{CopyGetters, Getters, Setters};
+use getset::{CopyGetters, Getters, MutGetters, Setters};
 use serde::{Deserialize, Serialize};
 use std::{
     collections::HashMap,
@@ -13,7 +13,17 @@ use std::{
 };
 
 #[derive(
-    Builder, Clone, CopyGetters, Debug, Deserialize, Eq, Getters, Setters, PartialEq, Serialize,
+    Builder,
+    Clone,
+    CopyGetters,
+    Debug,
+    Deserialize,
+    Eq,
+    Getters,
+    MutGetters,
+    Setters,
+    PartialEq,
+    Serialize,
 )]
 #[serde(rename_all = "camelCase")]
 #[builder(
@@ -57,7 +67,7 @@ pub struct ImageManifest {
     /// The final filesystem layout MUST match the result of applying
     /// the layers to an empty directory. The ownership, mode, and other
     /// attributes of the initial empty directory are unspecified.
-    #[getset(get = "pub", set = "pub")]
+    #[getset(get_mut = "pub", get = "pub", set = "pub")]
     layers: Vec<Descriptor>,
     /// This OPTIONAL property contains arbitrary metadata for the image
     /// manifest. This OPTIONAL property MUST use the annotation
@@ -164,11 +174,6 @@ impl ImageManifest {
     pub fn to_writer_pretty<W: Write>(&self, writer: &mut W) -> Result<()> {
         to_writer(&self, writer, true)
     }
-
-    /// Access the image layers mutably.
-    pub fn get_layers_mut(&mut self) -> &mut Vec<Descriptor> {
-        &mut self.layers
-    }
 }
 
 #[cfg(test)]
@@ -245,7 +250,7 @@ mod tests {
         let mut manifest = create_manifest();
         assert_eq!(manifest.layers().len(), 3);
         let layer_copy = manifest.layers()[0].clone();
-        manifest.get_layers_mut().push(layer_copy);
+        manifest.layers_mut().push(layer_copy);
         assert_eq!(manifest.layers().len(), 4);
     }
 ```